sql/agtype_operators.sql

```diff
@@ -53,3 +53,37 @@ CREATE OPERATOR <@ (
   RESTRICT = contsel,
   JOIN = contjoinsel
 );
+
```

Comment:
Please update this part once PR #1506 has been merged. These will need to be added to the upgrade file.